fluentlenium-spock/src/main/groovy/org/fluentlenium/adapter/spock/FluentSpecification.groovy

```diff
@@ -0,0 +1,56 @@
+package org.fluentlenium.adapter.spock
+
+import org.fluentlenium.adapter.FluentTestRunnerAdapter
+import org.fluentlenium.adapter.junit.FluentTestRule
+import org.junit.ClassRule
+import org.junit.Rule
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+import spock.lang.Specification
+
+class FluentSpecification extends Specification {
+
+    protected FluentSpecificationAdapter fl = new FluentSpecificationAdapter()
```

Comment:
I checked the project lombok page, and it seems like the @Delegate is move to experimental and they actually do not recommend using it anymore.

I did add a groovy delegate and it does seem to work. I have added a new commit with this change to the same PR.